distributed/scheduler.py

```diff
@@ -4683,53 +4683,48 @@ def handle_task_erred(self, key: str, stimulus_id: str, **msg) -> None:
         self.send_all(client_msgs, worker_msgs)
 
     def handle_missing_data(
-        self, key: str, errant_worker: str, stimulus_id: str, **kwargs
+        self, key: str, worker: str, errant_worker: str, stimulus_id: str
```

Comment:
For my clarification: how does `worker` get passed in? I don't see it as a field in `MissingDataMsg`.

Reply:
It's automatically added by ``self.batched_stream.send``

Reply:
I don't see logic for that anywhere in `BatchedSend.send`, or within the `_background_send` coroutine?
https://github.com/dask/distributed/blob/b8b45c6e641f298f465d47139f1fb297c4a5799a/distributed/batched.py#L131-L143

Reply:
It's here:
https://github.com/dask/distributed/blob/6e0fe58f9e0d15979187b3c790121d77cf743b05/distributed/scheduler.py#L4817